pennylane/ops/qubit/state_preparation.py

```diff
@@ -232,6 +232,9 @@ def state_vector(self, wire_order=None):
         return math.convert_like(ket, op_vector)
 
 
+QubitStateVector = StatePrep  # QSV is an alias for StatePrep
```

Comment:
One option that might make plugins better:
```suggestion
class QubitStateVector(StatePrep):
    pass
```

Reply:
Added 👍🏼